apartmentsbot/bot_handler.py

```diff
@@ -0,0 +1,117 @@
+from telebot import types
+from telebot.types import ReplyParameters
+from apartmentsbot.apartment_manager import ApartmentManager, MyStates
+
+
+class BotHandler:
+    def __init__(self, bot_instance):
+        self.bot = bot_instance
+        self.apartment_manager = ApartmentManager()
+
+    def start_help_message(self, message):
+        self.bot.send_message(
+            message.chat.id,
+            "Hello! I'm a bot for finding apartments in Belgrade or Novi Sad.\n"
```

Comment:
try not to store static values in your function code. This is both inefficient and impractical. This piece of data will be initialized each time a function is called (instead doing this once somewhere in another module). Also if you need to change a greeting message you have to search for it in another business logic code. And this can be confusing for you or someone else who is not familiar with your code at all

Reply:
Basically all the static values here and below, you can add them to the config file or create a single python file that will work as a config

Reply:
done